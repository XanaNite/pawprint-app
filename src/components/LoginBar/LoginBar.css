* {
    box-sizing: border-box;
}

.loginBar-container{
    text-align: center;
    border-top: 1px solid lightgray;
    padding: 20px;
    background: #214A77;
    position: fixed;
    left: 0;
    bottom: 0;
    width: 100%;
    color: white;
}

.loginBar-link{
    text-decoration: none;
    font-weight: bolder;
    color: white;
    text-shadow: 1px 1px gainsboro;
}

@media only screen and (min-width: 1000px) {
    /* tablets and desktop */
.loginBar-link{
    font-size: 22px;
}
}