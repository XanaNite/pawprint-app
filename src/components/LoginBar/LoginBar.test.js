import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoginBar from './LoginBar';

describe(`LoginBar component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders LoginBar by default', () => {
    const wrapper = shallow(<LoginBar />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the LoginBar given props', () => {
    const wrapper = shallow(<LoginBar {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
