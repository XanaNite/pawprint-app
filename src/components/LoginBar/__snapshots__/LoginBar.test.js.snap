// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBar component renders LoginBar by default 1`] = `
<div
  className="loginBar-container"
>
  <Link
    className="loginBar-link"
    to="/login"
  >
    Login
  </Link>
</div>
`;

exports[`LoginBar component renders the LoginBar given props 1`] = `
<div
  className="loginBar-container"
>
  <Link
    className="loginBar-link"
    to="/login"
  >
    Login
  </Link>
</div>
`;
